package/yast2-storage-ng.spec

```diff
@@ -101,4 +109,7 @@ rake test:unit
 %doc README.md
 %doc CONTRIBUTING.md
 
+%files -n yast2-partitioner-ng
```

Comment:
I would remove the "-ng" part here. There was never a yast2-partitioner package in the past, so no need to distinguish it.